### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

File: pandas/core/groupby/groupby.py [hunk 1]
```diff
@@ -3303,6 +3331,20 @@ def ohlc(self) -> DataFrame:
...
+        For Resampler:
+
+        >>> ser = pd.Series([1, 3, 2, 4, 3, 5],
+        ...                 index=pd.DatetimeIndex(['2023-01-01',
+        ...                                         '2023-01-10',
+        ...                                         '2023-01-15',
+        ...                                         '2023-02-01',
+        ...                                         '2023-02-10',
+        ...                                         '2023-02-15']))
+        >>> print(ser.resample('MS').ohlc())
```
Comment (MarcoGorelli) [inline pandas/core/groupby/groupby.py:3344]: I think we don't need the print statement? can be fixed up as part of a future PR anyway

File: pandas/core/resample.py [hunk 2]
```diff
@@ -1109,18 +1109,89 @@ def sum(
         *args,
         **kwargs,
     ):
+        """
+        Compute sum of group values.
+
+        Parameters
+        ----------
+        numeric_only : bool, default no
```
Comment (MarcoGorelli) [inline pandas/core/resample.py:1117]: default False?

File: pandas/core/resample.py [hunk 3]
```diff
@@ -1109,18 +1109,89 @@ def sum(
...
         min_count: int = 0,
         *args,
         **kwargs,
     ):
+        """
+        Compute prod of group values.
+
+        Parameters
+        ----------
+        numeric_only : bool, default no
```
Comment (MarcoGorelli) [inline pandas/core/resample.py:1164]: likewise

**Review by MarcoGorelli — COMMENTED** [review]
nice - couple of minor comments

File: pandas/core/groupby/groupby.py (hunk 1 above)
Comment (DeaMariaLeon) [reply pandas/core/groupby/groupby.py]: Thanks. (It looked weird in Jupyter without it)

File: pandas/core/resample.py (hunk 2 above)
Comment (DeaMariaLeon) [reply pandas/core/resample.py]: Thanks

File: pandas/core/resample.py (hunk 3 above)
Comment (DeaMariaLeon) [reply pandas/core/resample.py]: Done.

**Review by MarcoGorelli — APPROVED** [review]
nice one, thanks @DeaMariaLeon

**Pull request merged**